src/calculateRentalCost.js

```diff
@@ -1,10 +1,18 @@
-/**
+/*
  * @param {number} days
  *
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PriseDay = 40;
```

Comment:
Prefer using `const` for constants and follow UPPER_SNAKE_CASE naming convention for constant values. Rename `PriseDay` to `PRICE_PER_DAY`.